concordium-consensus/src/Concordium/Skov/Monad.hs

```diff
@@ -198,13 +198,20 @@ class (SkovQueryMonad m, TimeMonad m, MonadLogger m) => SkovMonad m where
     trustedFinalize :: FinalizationRecord -> m (Either UpdateResult (BlockPointerType m))
     -- |Handle a catch-up status message.
     handleCatchUpStatus :: CatchUpStatus -> Int -> m (Maybe ([(MessageType, ByteString)], CatchUpStatus), UpdateResult)
-    -- |Clean up the Skov state once it is shut down (i.e. a protocol update has
-    -- occurred). Returns a list of non-finalized transactions.
-    terminateSkov :: m [BlockItem]
+    -- |Clean up the Skov state upon a protocol update, removing all blocks that
+    -- are made obsolete by the protocol update. This should maintain all
+    -- invariants normally maintained by the Skov state, e.g., transaction table
+    -- invariants. occurred). Returns a list of non-finalized transactions.
```

Comment:
Where is the list?

Reply:
It is gone, and now also from the documentation.